src/Image.cc

```diff
@@ -880,7 +880,11 @@ cairo_status_t
 Image::loadJPEG(FILE *stream) {
   cairo_status_t status;
 
+#if defined(_MSC_VER)
+  if (false) { // Force using loadJPEGFromBuffer
+#else
   if (data_mode == DATA_IMAGE) { // Can lazily read in the JPEG.
```

Comment:
@LinusU To prevent repeating the buffer reading stuff https://github.com/Automattic/node-canvas/blob/master/src/Image.cc#L901-L911
Can of course refactor if you wish. Let me know.